src/Clock/Clock.tsx

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-console */
+import React from 'react';
+
+type Props = {
+  name: string
+};
+
+type State = {
+  today: Date,
+  clockName: string,
+};
+
+export class Clock extends React.Component<Props, State> {
+  state = {
+    today: new Date(),
+    clockName: this.props.name,
+  };
+
+  timerId = 0;
+
+  timerIdEverySec = 0;
+
+  componentDidMount(): void {
+    this.setState({ today: new Date() });
+
+    this.timerIdEverySec = window.setInterval(() => {
+      this.setState({ today: new Date() });
+      console.info(new Date().toUTCString().slice(-12, -4));
+    }, 1000);
+
+    this.timerId = window.setInterval(() => {
+      this.setState({ clockName: this.props.name });
+    }, 3300);
```

Comment:
You have 2 states for 1 name control `clockName` only in App component
 https://mate-parttime.slack.com/archives/C03UQS59NRY/p1685986512064139?thread_ts=1685983236.290469&cid=C03UQS59NRY